extensions/wikia/HTTPSSupport/tests/HTTPSSupportHooksTest.php

```diff
@@ -86,4 +90,98 @@ public function onBeforeResourceLoaderCSSMinifierDataProvider() {
 			]
 		];
 	}
+	/**
+	 * @param string $url
+	 * @param string $environment
+	 * @param string $expectedResult
+	 *
+	 * @dataProvider onLinkerMakeExternalLinkDataProvider
+	 */
+	public function testOnLinkerMakeExternalLink( $url, $environment, $expectedResult ) {
+
+		$exampleString = '';
+		$exampleBool = false;
+		$exampleArray = [];
+
+		$this->mockEnvironment( $environment );
+		HTTPSSupportHooks::onLinkerMakeExternalLink($url, $exampleString, $exampleBool, $exampleArray);
+		$this->unsetGlobals();
+		$this->assertEquals( $expectedResult, $url );
+
+	}
+
+	public function onLinkerMakeExternalLinkDataProvider() {
+		return [
+			[
+				'$url' => 'http://www.example.com',
+				'$environment' => WIKIA_ENV_PROD,
+				'$expectedResult' => 'http://www.example.com',
+			],
+			[
+				'$url' => 'https://www.example.com' ,
```

Comment:
just to make sure only host is processed, you may add a tricky case with `https://www.example.com/www.fandom.com/aaa`